packages/config/config/devices/0x0346/panic_button_gen2.json

```diff
@@ -0,0 +1,71 @@
+{
+	"manufacturer": "Ring",
+	"manufacturerId": "0x0346",
+	"label": "Ring Alarm Panic Button",
+	"description": "Panic Button (2nd Gen)",
```

Comment:
This will cause some applications to display the device as
> Ring Ring Alarm Panic Button Panic Button (2nd Gen)

Do you know the model number by any chance, e.g. like on the contact sensor:
```
	"manufacturer": "Ring",
	"manufacturerId": "0x0346",
	"label": "4SD2SZ-0EN0",
	"description": "Contact Sensor Gen2",
```

Reply:
I'll ask.